### PR TITLE
fix: remove broken Netlify logo

### PR DESCRIPTION
**Pull request by lloesche**
# Description

Logo is broken, this removes it.

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -258,13 +258,6 @@ const config = {
             ],
           },
         ],
-        logo: {
-          alt: 'Deploys by Netlify',
-          src: 'https://netlify.com/img/global/badges/netlify-color-accent.svg',
-          width: 114,
-          height: 51,
-          href: 'https://netlify.com',
-        },
```
Comment (TheCatLady) [inline docusaurus.config.js:267]: We can't just remove it, as it's a requirement for the Netlify Open Source plan.  Let's just leave it and wait for it to be fixed.

File: docusaurus.config.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const a11yEmoji = require('@fec/remark-a11y-emoji');
 const config = {
   title: 'Resoto by Some Engineering Inc.',
   tagline:
-    'Find leaky resources, manage quota limits, detect drift, and clean up!',
+    'Automate tedious infrastructure tasks, remarkably fast!',
```
Comment (TheCatLady) [inline docusaurus.config.js:10]:
```suggestion
    'Effortlessly automate tedious infrastructure tasks!',
```
I'm not sure if "remarkably fast" is accurate, and fast isn't always good.

File: docusaurus.config.js (hunk 1 above)
Comment (lloesche) [reply docusaurus.config.js:267]: We'll just pay for Netlify. I already put a card behind it back then.

File: docusaurus.config.js (hunk 2 above)
Comment (lloesche) [reply docusaurus.config.js:10]: It's the tag line we're using right now. Can change it later.

**Pull request merged**